### DISCUSSION TITLE
Need list of all prometheus.io annotations supported for Kubernetes environment

### DISCUSSION THREAD

**Discussion by arijitr-citrix** [prometheus/prometheus#13611] [General]
Hi Team,

I am looking to customise the metrics endpoint on my applications and would like to send these customisations directly to prometheus without changes in the prometheus.yml file. For example, we can have the custom path using the annotation:
prometheus.io/path

In the same way, I would like to send the certain authentication specific information driven by an annotation. Would like to know if there is such annotation that can help our requirement. On top of that there are more customisations needed and it would help if I can get the list of supported annotations. A standard google search did not help.

TIA.
Arijit

**Comment by rjchicago** [comment]
Similar request - I was also looking for support for a `Header` annotation. I don't see that in docs anywhere.

Example
``` yaml
prometheus.io/header: 'Cookie: key=value'
```
